config/main.py

```diff
@@ -185,6 +193,8 @@ def run_command(command, display_cmd=False, ignore_error=False):
 
     if proc.returncode != 0 and not ignore_error:
         sys.exit(proc.returncode)
+    else:
+        return (out, err)
```

Comment:
Please update the docstring to note that upon success the function returns a tuple containing the contents of stdout and stderr.